app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -584,6 +646,33 @@ class BrowserTabFragment : Fragment(), FindListener, CoroutineScope, DaxDialogLi
         }
     }
 
+    private fun checkSystemLocationPermission(domain: String, deniedForever: Boolean) {
+        if (ContextCompat.checkSelfPermission(requireActivity(), Manifest.permission.ACCESS_FINE_LOCATION) != PackageManager.PERMISSION_GRANTED) {
```

Comment:
We ask for both `ACCESS_FINE_LOCATION` and `ACCESS_COARSE_LOCATION`, but only check the result for `ACCESS_FINE_LOCATION`. Does this matter?